coral/src/app/features/topics/details/components/PromotionBanner.tsx

```diff
@@ -14,6 +14,8 @@ interface PromotionBannerProps {
   type: "schema" | "topic";
   promoteElement: ReactElement;
   hasOpenRequest: boolean;
+  hasError: boolean;
+  errorMessage: string;
```

Comment:
"errorMessage" in this case is optional because for example `TopicPromotionBanner` does not have the need to show an error message. And if we use update this promotion banner to be more generic (e.g. also be used for claim) there may be cases where we need to be able to show an error here, but in most cases the "button" links to a form, which has it's own error handling. 

>"Standardized mutation error handling"? 

We could do that, but for this component I don't think that we should assume that the error shown here is a mutation error. It is currently in the one use case, but it could also be a different kind of error. And tying or components too closely to structures of libs we're using may restrict us a bit. What if we want to / need to get rid of react-query -> or console wants to use components from us and they don't use react-query.